packages/web3-utils/CHANGELOG.md

```diff
@@ -173,4 +173,9 @@ Documentation:
 
 ### Added
 
--   Add `isContractInitOptions` method (#6455)
\ No newline at end of file
+-   `SocketProvider` now contains public function `getPendingRequestQueueSize`, `getSentRequestsQueueSize` and `clearQueues` (#6451)
+-   Added `safeDisconnect` as a `SocketProvider` method to disconnect only when request queue size and send request queue size is 0 (#6451) 
+
+### Fixed
+
+-   Add `isContractInitOptions` method (#6455)
```

Comment:
I think this should be still under `Added` section. But I noticed that the PR number was wrong (was 6455 but should be 6555), right?

```suggestion
-   Add `isContractInitOptions` method (#6555)

### Fixed
```